example/test.py

```diff
@@ -102,26 +95,33 @@ def create_django_bootstrap_container(
         name, DJANGO_BOOTSTRAP_IMAGE, **kwargs)
 
 
-class TestWeb(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.web_container = create_django_bootstrap_container(
-            docker_helper, 'web')
-        docker_helper.start_container(cls.web_container, r'Booting worker')
+@pytest.fixture(scope='class')
+def web_container(docker_helper, db_container, amqp_container):
```

Comment:
The db and amqp containers have to be passed in even if they are not used to ensure proper ordering of the container starts.